docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
Why do we need to perform such change? I have never seen such pattern for importing React. That's hiding something.